MCLOUD-7604: 'use_stale_cache' option use (#8672)
magento/devdocs: src/guides/v2.4/config-guide/cache/two-level-cache.md

## Configuration example

> Put this into the `app/etc/env.php` file or replace existed cache section.

Reviewer: ```suggestion
Put this into the `app/etc/env.php` file or replace the existing cache section.
```

```php
'cache' => [
    'frontend' => [
        'default' => [
            'backend' => '\\Magento\\Framework\\Cache\\Backend\\RemoteSynchronizedCache',
            'backend_options' => [
                'remote_backend' => '\\Magento\\Framework\\Cache\\Backend\\Redis',
                'remote_backend_options' => [
                    'persistent' => 0,
                    'server' => 'localhost',
                    'database' => '0',
                    'port' => '6379',
                    'password' => '',
                    'compress_data' => '1',
                ],
                'local_backend' => 'Cm_Cache_Backend_File',
                'local_backend_options' => [
                    'cache_dir' => '/dev/shm/'
                ],
                'use_stale_cache' => false,
            ],
            'frontend_options' => [
                'write_control' => false,
            ],
        ]
    ],
    'type' => [
        'default' => ['frontend' => 'default'],
    ],
],
```

Where:

-  `backend` is the L2 cache implementation.

> -  `backend_options` are the L2 cache configuration.

Reviewer: ```suggestion
-  `backend_options` is the L2 cache configuration.
```

   -  `remote_backend` is the remote cache implementation: Redis or MySQL.

>    -  `remote_backend_options` are the remote cache configuration.

Reviewer: ```suggestion
   -  `remote_backend_options` is the remote cache configuration.
```

   -  `local_backend` is the local cache implementation: `Cm_Cache_Backend_File` or the APC adapter.

>    -  `local_backend_options` are the local cache configuration.

Reviewer: ```suggestion
   -  `local_backend_options` is the local cache configuration.
```

>       -  `cache_dir` __a file cache specific option.__ Is a directory where the local cache will be stored.

Reviewer: ```suggestion
      -  `cache_dir` __a file cache specific option.__ Is the directory where the local cache will be stored.
```

   -  `use_stale_cache` is a flag that enable\disables the stale cache mechanism.

> We recommend the use of Redis for remote caching - `\Magento\Framework\Cache\Backend\Redis`, and the File cache implementation - `Cm_Cache_Backend_File` as the local cache storing data in shared memory - `'local_backend_options' => ['cache_dir' => '/dev/shm/']`.

Reviewer: ```suggestion
We recommend the use of Redis for remote caching - `\Magento\Framework\Cache\Backend\Redis`, and the File cache implementation - `Cm_Cache_Backend_File` as the local cache for storing data in shared memory - `'local_backend_options' => ['cache_dir' => '/dev/shm/']`.
```

> We recommend the use of Redis for remote caching - `\Magento\Framework\Cache\Backend\Redis`, and the File cache implementation - `Cm_Cache_Backend_File` as the local cache for storing data in shared memory - `'local_backend_options' => ['cache_dir' => '/dev/shm/']`.

Reviewer 2: ```suggestion
We recommend using Redis for remote caching (`\Magento\Framework\Cache\Backend\Redis`) and the `Cm_Cache_Backend_File` implementation for local caching of data in shared memory, using `'local_backend_options' => ['cache_dir' => '/dev/shm/']`.
```
